docker/compose-controller-spark-sql-single.yaml

```diff
@@ -94,7 +96,11 @@ services:
       start_period: 10s
       timeout: 60s
 
+
+volumes:
+  spark_vol_single:
```

Comment:
Should we call this single even the file name for that matter? or could we call suffix this with `simple` or something like that?

Reply:
`single` in this case is an indication that we only have one spark process. I can call it `spark_vol_thrift` if you want but `simple` is probably confusing as the other compose file is arguably simple too :-)

Reply:
sure, not an issue as of now :)